Frends.Community.LDAP.Tests/UnitTests.cs

```diff
@@ -1,269 +1,291 @@
+                Password = Environment.GetEnvironmentVariable("HiQAzureADTestPassword", EnvironmentVariableTarget.User),
+            };
+
+            _passConnection = new PasswordParameters
+            {
+                AdServer = Environment.GetEnvironmentVariable("HiQAzureADTestAddress", EnvironmentVariableTarget.User),
+                AdContainer = "DC=FRENDSTest01",
+                Username = Environment.GetEnvironmentVariable("HiQAzureADTestUser", EnvironmentVariableTarget.User),
+                Password = Environment.GetEnvironmentVariable("HiQAzureADTestPassword", EnvironmentVariableTarget.User),
+
+                // UserPrincipalName not confirmed to exist in test AD - TO DO!
+                UserPrincipalName = "Matti.Meikalainen@testi.fi",
+
+                // HiQAzureADTestUserNewPassword has not yet been set - TO DO!
+                NewPassword = Environment.GetEnvironmentVariable("HiQAzureADTestUserNewPassword", EnvironmentVariableTarget.User),
+                ContextOptionFlags = new []
+                    {
+                    new Context { ContextOptionFlag = ContextOption.SimpleBind, Value = true },
+                    new Context { ContextOptionFlag = ContextOption.Signing, Value = true },
+                    new Context { ContextOptionFlag = ContextOption.ServerBind , Value = true }
+                    }
+            };
+
+        }
+
+        [Test, Order(1)]
+        public void ShouldCreateUser()
+        {
+            var user = new CreateADuser
+            {
+                CN = _user,
+                Path = _path
+            };
+            var attributes = new List<EntryAttribute> { new EntryAttribute() { Attribute = AdUserAttribute.givenName, Value = "Matti", DataType = AttributeType.String } };
+            user.OtherAttributes = attributes.ToArray();
+
+            var flags = new List<ADFlag>
+            {   
+                new ADFlag {FlagType = ADFlagType.ADS_UF_ACCOUNTDISABLE, Value = false},
+                new ADFlag {FlagType = ADFlagType.ADS_UF_NORMAL_ACCOUNT, Value = true}
+            };
+            user.ADFlags = flags.ToArray();
+
+            var e = new AD_CreateUserProperties()
+            {
+                NewPassword = "",
+                SetPassword = false
+            };
+            var result = LdapActiveDirectoryOperations.AD_CreateUser(_connection, user, e);
+            Assert.AreEqual(result.OperationSuccessful, true);
+        }
+
+
+        [Test, Order(2)]
+        public void ShouldFetchUser()
+        {
+            var e = new AD_FetchObjectProperties()
+            {
+                Filter = "(&(objectClass=user)(cn=" + _user + "))",
+                Path = _path
+            };
+            var u = LdapActiveDirectoryOperations.AD_FetchObjects(_connection, e);
+            var result = u[0].GetProperty("cn");
+            Assert.AreEqual(result[0], _user);

```

Comment:
Please flip the parameters around, since the first parameter should be the expected value instead of real value.